src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/ParseDateTimeArgs.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml;
+
+namespace System.ServiceModel.Syndication
+{
+    public struct ParseDateTimeArgs
+    {
+        public string DateTimeString { get; set; }
+        public XmlQualifiedName ElemntQualifiedName { get; set; }
```

Comment:
Should this be publicly settable? Looking through over Args types in the framework, the setter is only there if you expect developers to set that value. We should have an internal constructor taking the parameters and remove the public setter.